exporter/dynatraceexporter/config/config.go

```diff
@@ -45,28 +46,28 @@ type Config struct {
 	Prefix string `mapstructure:"prefix"`
 }
 
-// Sanitize ensures an API token has been provided
-func (c *Config) Sanitize() error {
-	c.APIToken = strings.TrimSpace(c.APIToken)
-
-	if c.APIToken == "" {
-		return errors.New("missing api_token")
+// ValidateAndConfigureHTTPClientSettings validates the configuration and sets default values
+func (c *Config) ValidateAndConfigureHTTPClientSettings() error {
+	if c.HTTPClientSettings.Headers == nil {
+		c.HTTPClientSettings.Headers = make(map[string]string)
 	}
+	c.APIToken = strings.TrimSpace(c.APIToken)
 
 	if c.Endpoint == "" {
-		return errors.New("missing endpoint")
+		c.Endpoint = apiconstants.GetDefaultOneAgentEndpoint()
+	} else {
+		if c.APIToken == "" {
+			return errors.New("APIToken is required if Endpoint is provided")
```

Comment:
More user friendly if you use the YAML name?
```suggestion
			return errors.New("api_token is required if Endpoint is provided")
```